pihole

```diff
@@ -98,8 +99,25 @@ versionFunc() {
   exit 0
 }
 
+# Get PID of main pihole-FTL process
+getFTLPID() {
+  local pid
+
+  if [ -s "${FTL_PID_FILE}" ]; then
+    # -s: FILE exists and has a size greater than zero
+    pid="$(<"$FTL_PID_FILE")"
+    # Exploit prevention: unset the variable if there is malicious content
+    # Verify that the value read from the file is numeric
+    [[ "$pid" =~ [^[:digit:]] ]] && pid=
```

Comment:
Do you want to set `pid` to empty or actually `unset pid`?

Reply:
I don't care as long as it trips `${FOO:=val}`. Should happen both for unset and empty.

Reply:
`bash` style would be to use `unset`. `c` style would be to set it to nothing and let garbage collection handle it. 

Can we use `unset` for a purely stylistic reason?

Reply:
```suggestion
    [[ "$pid" =~ [^[:digit:]] ]] && unset pid
```